modin/pandas/series.py

```diff
@@ -946,7 +951,8 @@ def explode(self, ignore_index: bool = False):  # noqa: PR01, RT01, D200
         Transform each element of a list-like to a row.
         """
         return super(Series, self).explode(
-            "__reduced__" if self.name is None else self.name, ignore_index=ignore_index
+            MODIN_UNNAMED_SERIES_LABEL if self.name is None else self.name,
+            ignore_index=ignore_index,
```

Comment:
```suggestion
            self.name or MODIN_UNNAMED_SERIES_LABEL, ignore_index=ignore_index,
```

Reply:
as above, this is incorrect when result.name == ""